handler/src/test/java/io/netty/handler/ssl/OpenSslEngineTest.java

```diff
@@ -568,8 +568,8 @@ private void testWrapDstBigEnough(SSLEngine engine, int srcLen) throws SSLExcept
         assertFalse(src.hasRemaining());
     }
 
-    @Test(expected = IllegalArgumentException.class)
-    public void testSNIMatchersThrows() throws Exception {
+    @Test
+    public void testSNIMatchersNotThrows() throws Exception {
```

Comment:
nit: `testSNIMatchersDoNotThrow` or even better, since you don't need the `test` prefix with junit4: `sniMatchersShouldNotThrow`